ui/src/StateContainer.js

```diff
@@ -67,6 +72,34 @@ export class StateContainer extends LitElement {
         ).shadowRoot;
     }
 
+    /**
+     * Parses the URL for relevant project-related details. 
+     */
+    setProjectDetails() {
+        this.projectId = this.urlService.getProjectId();
```

Comment:
Just to confirm, projectId and userLang and vendorId would not be set on the cookie? Only user entered info and the userId?

Reply:
Yes--right now only userId and entered info would be saved in the cookie (as the project related stuff will change with every new link & can be easily parsed from the URL)